### DISCUSSION TITLE
another math issue

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#19480] [Other Feature Feedback, Questions, & Ideas] [answered]
The following math is incorrectly rendered:
```markdow
When $a \ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$ and they are
```

![screenshot](https://user-images.githubusercontent.com/181628/175812775-b7da841f-1a80-4c21-ad84-4c9a65d47b8d.png)

When $a \ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$ and they are

**Comment by melvincwng** [comment]
@nschloe Hi Nico, you can report such issues over here to [GitHub Bug Report](https://support.github.com/contact/bug-report).

Hope this is helpful.

**Comment by crvdgc** [comment]
I encountered the exact same issue and wrote up a showcase and my analysis of it: https://gist.github.com/crvdgc/8278d5fbadb6eec0c47db22552f20f6f.

I'll submit this to the bug report as well.

**Comment by nschloe** [comment, marked as answer]
This has been fixed.
